src/main/java/Deadline.java

```diff
@@ -0,0 +1,28 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class representation of Deadline
+ */
+public class Deadline extends Task{
+    protected LocalDateTime dL;
+
+    /**
+     * Deadline Constructor
+     * @param description
+     * @param dL
+     */
+    public Deadline(String description, LocalDateTime dL) {
+        super(description);
+        this.dL = dL;
+    }
+
+    /**
+     * Deadline's overriden toString method
+     * @return String representation of Deadline
+     */
+    public String toString() {
```

Comment:
Perhaps you could shift a part of your toString() method to your Task class as ToDo, Deadline and Event's toString() methods all share some similarities (in terms of this.getStatusIcon() and this.description). Subsequently, you can call super.toString() to shorten your code! :)